sqlmesh/core/snapshot/definition.py

```diff
@@ -299,6 +311,20 @@ class SnapshotTableInfo(PydanticModel, SnapshotInfoMixin, frozen=True):
     kind_name: t.Optional[ModelKindName] = None
     node_type_: NodeType = Field(default=NodeType.MODEL, alias="node_type")
 
+    REPR_FIELDS = (
+        "name",
+        "fingerprint",
+        "version",
+        "physical_schema",
+        "change_category",
+        "kind_name",
+    )
+
+    def __str__(self) -> str:
+        args = ", ".join(f"{k}: {getattr(self, k)}" for k in self.REPR_FIELDS)
+        temp_version = f", {self.temp_version}" if self.temp_version else ""
+        return f"SnapshotInfo<{args}{temp_version}>"
```

Comment:
We should use the original class name. `SnapshotInfo` refers to an entity which doesn't exist